src/main/java/uk/gov/companieshouse/insolvency/data/converter/OffsetDateTimeReader.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.insolvency.data.converter;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+
+import org.springframework.core.convert.converter.Converter;
+
+public class OffsetDateTimeReader implements Converter<Date, OffsetDateTime> {
```

Comment:
Cosmetic: Maybe ***ReadConverter name be more appropriate and follows existing pattern in this package